### DISCUSSION TITLE
Calling custom actions

### DISCUSSION THREAD

**Discussion by lodmfjord** [ojkelly/yarn.build#104] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Sometimes it is great to check dependency and everyone that depends on it – typecheck, linter etc. 

**Describe the solution you'd like**
Being able to create custom actions. 

**Describe alternatives you've considered**
Using wsrun — which is the current way I am doing it.

**Comment by ojkelly** [comment, marked as answer]
Have you tried `yarn build -c scriptName`? Where `scriptName` is a script present in all your workspaces?

Or would you more like the ability to run an arbitrary script, but in the utilising the build cache and topology of your dependencies?

Anytime you run `yarn build` where it's just `build` or `test` or a `-c action` a cache is kept for each workspace and package, so we know if we need to re run it.

In that sense, `yarn build` is not really designed for ad hoc scripts, but those that are key to your dev/deploy loop.

**Comment by lodmfjord** [comment]
Thanks, that suits my need. I will close this issue.
